service/charon.service

```diff
@@ -11,6 +11,7 @@ BusName=nl.ultimaker.charon
 User=ultimaker
 Type=simple
 Restart=always
+SyslogIdentifier=Charon
```

Comment:
I would consider adding these 2 settings as well, since the `SyslogIdentifier` only applies if logging to the journal:
`StandardOutput=journal`
`StandardError=journal`